capstone-frontend/src/components/LeftSideBar.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css';
+
+class LeftSideBar extends Component {
```

Comment:
nit: This can be a stateless functional component.

Reply:
Fixed. Are there benefits to writing them as stateless functions, or is it just for readability (eg. viewers know that it's just a component for display)?